src/test/scala/uk/gov/hmrc/govukfrontend/views/layouts/twoThirdsMainContentSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.govukfrontend.views.layouts
+
+import play.twirl.api.HtmlFormat
+import uk.gov.hmrc.govukfrontend.views.TemplateUnitSpec
+import uk.gov.hmrc.govukfrontend.views.html.components.TwoThirdsMainContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.maincontent.MainContent
+
+import scala.util.Try
+
+class twoThirdsMainContentSpec extends TemplateUnitSpec[MainContent]("twoThirdsMainContent") {
```

Comment:
I wonder if we could just make this a regular unit test suite not extending TemplateUnitSpec, this would mean we wouldn't need the MainContent viewmodel and the static test fixtures in additional-fixtures?